### ISSUE TITLE
'content' does not display in universal viewer

### ISSUE THREAD

**Issue by lightgivener** [pulibrary/cicognara-catalogo#137]
Hey @tpendragon and @escowles,

I talked with Alexandra in Heidelberg the other day and she curiously could not see the 'contents' foldout section in her Universal Viewer that holds the index struct-map and thumbnails overview. Since I had the problem that the Universal Viewer only displayed as a black box for me for months which Trey could help me out with (third party cookies were blocked) I was hoping you would might have an idea how to fix.

Also we should probably note all display problems and fixes for a possible 'help' section as I guess Alexandra and I are not the only ones with viewer troubles.

Thanks!

![screenshot_DCL_universal_Viewer_macairbook](https://user-images.githubusercontent.com/4511569/84680187-961e1180-af32-11ea-85c5-4677b8aa340f.png)

**Comment by tpendragon** [comment]
This is an unfortunate bug with UV when your window's too small.

**Comment by lightgivener** [comment]
hm but she does see the '<<more information' fold-out section on the right? And for me it is either none when the window is too small or both when i widen the viewer.

**Comment by tpendragon** [comment]
Yeah, the more information always show. If the screen's too small there's no way to get to the table of contents. In short, thanks for the bug report, if there are commonly used screens at this size we'll have to find a way to fix it in UV.
